watchdog/map.jinja

```diff
@@ -0,0 +1,11 @@
+{% set server = salt['grains.filter_by']({
+    'HP': {
+      'kernel_module': 'hpwdt',
```

Comment:
No, this should be:
```
kernel: {
  'module': 'hpwdt',
  'parameter': {}
}
```

and instead of QEMU use `default` to have softdog on all unrecognized platforms (unless override in metadata)

Reply:
I agree with the first part, but not with the second one. 
QEMU driver should not be the "default".
The default should be "no driver", because if you will install the "qemu watchdog kernel module" to some unknown hardware server - it may cause error or unexpected behavior.

Reply:
ok

Reply:
Clarified...